autorest/codegen/serializers/__init__.py

```diff
@@ -29,6 +30,31 @@
     "JinjaSerializer",
 ]
 
+def _get_environment(template_path: str) -> Environment:
+    return Environment(loader=PackageLoader("autorest.codegen", template_path))
+
+
+def _build_package_render_parameters(code_model: CodeModel) -> Any:
+    parameters = {}
+    parameters["package_name"] = code_model.options["package_name"]
+    parameters["package_version"] = code_model.options["package_version"]
+    parameters["namespace"] = code_model.options["namespace"]
+    folder_list = parameters["package_name"].split('-')
+    parameters["folder_first"] = folder_list[0]
+    parameters["folder_second"] = folder_list[1]
+    parameters["test_prefix"] = folder_list[-1]
+    parameters["date_to_release"] = time.strftime('%Y-%m-%d', time.localtime())
```

Comment:
I feel that hardcoding date to release to the time when the script is run isn't the msot flexible for users, should we add another flag for this?

Reply:
Mgmt also has similar problem. The current solution is that we help to configure it with latest date. And users could change it when release

Reply:
Mgmt also has similar problem. The current solution is to fill it with latest time and users could change it with actual release time by themselves when release.